apps/frontend/src/components/DummyData/DummyData.tsx

```diff
@@ -0,0 +1,65 @@
+'use client';
+import { useInfiniteQuery } from '@tanstack/react-query';
+import { dummyDataApi } from 'api-client';
+import { useEffect } from 'react';
+
+import { BrowseProfileCard } from '../cards/BrowseProfileCard';
+
+import type { DummyDataDto } from 'api-client';
+
+export default function DummyData() {
+  const limit = 10;
+
+  const fetchProfiles = async () => {
+    const { data } = await dummyDataApi.getData({ limit }).throwOnError();
+    return data as DummyDataDto[];
+  };
+
+  const { data, error, fetchNextPage, hasNextPage, isFetchingNextPage } = useInfiniteQuery({
+    queryKey: ['profiles'],
+    queryFn: fetchProfiles,
+    initialPageParam: 0,
+    getNextPageParam: (lastPage, allPages) => {
+      const nextPage = lastPage.length === limit ? allPages.length * limit : undefined;
+      return nextPage;
+    },
+  });
+
+  if (error) {
+    throw error;
+  }
+
+  useEffect(() => {
+    async function handleScroll() {
```

Comment:
wrap `handleScroll` into a throttle function. 100ms delay will be ok as a default value, but provide an ability to change it if needed in `useScrollbar` hook